apps/testingaccessibility/bin/seed

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Description:
+#   Seeds the database.
+# Usage:
+#   bin/seed
+
+product='testing-accessibility'
+branch='next-steps'
```

Comment:
this should be set to another or variable database

next-steps is already seeded!

Reply:
“database create branch $name” would be great

Reply:
Ended up going with `pscale branch switch $planetscale_branch --database $database --create`.

That way it just switches if the branch exists but creates it if it doesn't exist.

It also derives the Planetscale DB name from the Git branch.

Reply:
should this be removed then because it's still hard coded to my branch in this script from what I can tell

stuff above looks good

Reply:
Good catch. Forgot to adjust that one.